modules/ROOT/pages/salesforce/salesforce-connector-100-config-topics.adoc

```diff
@@ -266,33 +257,23 @@ The JSON received as a response from the push event operation looks something li
 [[batchdata]]
 == Load Data in Batches
 
-The Salesforce Bulk API loads batches of your organization's data into Salesforce.
+The Salesforce Bulk API loads batches of your organization's data into Salesforce. Salesforce Connector provides the Create and Create Bulk operations for working with the Bulk API.
 
-The Salesforce connector provides the Create and Create Bulk operations for working
-with the Bulk API.
+For all bulk operations, Salesforce handles the creation process in the background. The connector replies with a BatchInfo object, which contains the ID of the batch and the ID of the job it creates to upload the objects.
 
-For all bulk operations, Salesforce handles the creation process in the background, so the connector does not reply with a collection of SaveResults because it does not have them yet. Instead, the connector replies with a BatchInfo object, which contains the ID of the batch and the ID of the job it creates to upload those objects.
+// Find out which objects this refers to
```

Comment:
Any type of Salesforce entity: Account, Contact, Lead and all the other available - even custom objects

Reply:
Used the term "batched objects" to make this clearer.